openspec/changes/dallay-413-bulk-scheduling/apply-progress.md

```
@@ -0,0 +1,104 @@
+# Apply Progress — DALLAY-413 Bulk Scheduling — PR1+PR2+PR3 (complete)
+
+## Branch / Worktree
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)